library/ssl_tls12_server.c

```diff
@@ -3233,6 +3241,12 @@ static int ssl_prepare_server_key_exchange( mbedtls_ssl_context *ssl,
          */
         if( md_alg != MBEDTLS_MD_NONE )
         {
+            if( dig_signed == NULL )
```

Comment:
It's too late to check this: if `dig_signed` is null then the initialization of `dig_signed_len` above already invoked undefined behavior.